### PR TITLE
CNDE-1403: Post-processing integration for LAB100 and LAB101 datamarts

### PR DESCRIPTION
**Pull request by sveselev**
## Notes

For `nrt_observation` topic post-processing: add call for `sp_lab100_datamart_postprocessing` and  `sp_lab101_datamart_postprocessing` stored procs.

## JIRA

- **Related story**: [CNDE-1403](https://cdc-nbs.atlassian.net/browse/CNDE-1403)

## Checklist

- [x] PR focuses on a single story.
- [x] New unit tests added and ensured they pass.
- [x] Service has been tested in local and it works as expected.
- [ ] Documentation has been updated for this code change (if needed).
- [x] Code follows the Java Coding Conventions (https://www.oracle.com/java/technologies/javase/codeconventions-programmingpractices.html).

## Types of changes

What types of changes does this PR introduces?

- [ ] Bugfix
- [x] New feature
- [ ] Breaking change

## Testing

- [x] Does this PR has >90% code coverage?
- [ ] Is the screenshot attached for code coverage?
- [x] Does the `gradle build` pass in your local? 
- [ ] Is the `gradle build` logs attached?

### CODE REVIEW

**Review by upasanapattnaik-eq — APPROVED** [review]
LGTM!

File: post-processing-service/src/main/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingService.java [hunk 1]
```diff
@@ -312,13 +317,14 @@ private String extractValFromMessage(String topic, String payload) {
                 }
             } else if (topic.endsWith(Entity.OBSERVATION.getName())) {
                 String domainCd = objectMapper.readTree(payload).get(PAYLOAD).path("obs_domain_cd_st_1").asText();
-                String ctrlCd = objectMapper.readTree(payload).get(PAYLOAD).path("ctrl_cd_display_form").asText();
+                String ctrlCd = Optional.ofNullable(objectMapper.readTree(payload).get("ctrl_cd_display_form"))
+                        .filter(node -> !node.isNull()).map(JsonNode::asText).orElse(null);
```
Comment (alog-enquizit) [inline post-processing-service/src/main/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingService.java:321]: Suggestion:  you can consider using `ObjectUtils.isEmpty` which has enhanced null pointer chekcks.

**Review by alog-enquizit — APPROVED** [review]
lgtm 👍

**Comment by alog-enquizit** [comment]
@sveselev Looks like some unit tests are failing. Please fix them before merging.

File: post-processing-service/src/main/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingService.java (hunk 1 above)
Comment (sveselev) [reply post-processing-service/src/main/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingService.java:321]: > Suggestion: you can consider using `ObjectUtils.isEmpty` which has enhanced null pointer chekcks.

Unfortunately in this case the node is not null, it's NullNode class.

**Pull request merged**